gp-includes/warnings.php

```diff
@@ -153,6 +153,16 @@ class GP_Builtin_Translation_Warnings {
 	 */
 	public $length_exclude_languages = array( 'art-xemoji', 'ja', 'ko', 'zh', 'zh-hk', 'zh-cn', 'zh-sg', 'zh-tw' );
 
+	/**
+	 * List of domains with allowed changes to their own subdomains
+	 *
+	 * @since 3.0.0
+	 * @access public
+	 *
+	 * @var array
+	 */
+	public $allowed_domain_changes = array();
```

Comment:
This should probably contain the domains from `GP_Wporg_Translation_Warnings` as default.

Reply:
Done.